easybuild/easyconfigs/p/PLINK/PLINK-2.00a3.7-GCC-12.3.0.eb

```diff
@@ -0,0 +1,39 @@
+# Author: Ben Czaja (SURF) based on orginal from Jasper Grimm (UoY)
+
+easyblock = 'MakeCp'
+
+name = 'PLINK'
+version = '2.00a3.7'
+
+homepage = 'https://www.cog-genomics.org/plink/2.0/'
+description = 'Whole-genome association analysis toolset'
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/chrchang/plink-ng/archive']
+sources = ['v%(version)s.tar.gz']
+checksums = ['145717350205f5562a01292a0fcbd617b7500758f20cad0393c7cc54665a614e']
+
+dependencies = [
+    ('FlexiBLAS', '3.3.1'),
+    ('libdeflate', '1.18'),
+    ('zstd', '1.5.5'),
+]
+
+start_dir = '2.0'
+
+# ignore suggested Makefile in build_dynamic, the dev Makefile is better
+buildopts = 'BLASFLAGS="-lflexiblas" BLASFLAGS64="-lflexiblas"'
+buildopts += ' LDFLAGS="$LIBS -ldeflate -lz -lzstd" ZLIB="" CSRC="include/SFMT.c"'
```

Comment:
Since you have -lz here zlib should also be explicitly mentioned in the dependencies.